src/formio/templates/signature.ejs

```diff
@@ -9,7 +9,10 @@
     class="btn btn-sm btn-light signature-pad-refresh utrecht-button utrecht-button--html-button utrecht-button--subtle"
     ref="refresh"
   >
-    <i class="{{ctx.iconClass('refresh')}}"></i>
+    <i class="{{ctx.iconClass('refresh')}}">
+      <!-- For accessibility a Button element must contain text content -->
+      <span class="sr-only">Refresh</span>
```

Comment:
pass this through `{{ctx.t('Refresh')}}` to support translations/internationalization

Reply:
@sergei-maertens Should _<span class="sr-only">_ then become _<span class="{{ctx.iconClass('sr-only')}}">Refresh</span>_ here? (Since that is where the text is situated?)
Or should the end-result simply be {{ctx.**iconClass.t(**'Refresh')}} that surrounds all?

Reply:
nah, just stick to hardcoded `sr-only` here. Formio supports other icon libraries than font-awesome (technically), but we're not using that feature/functionality.

The end result should thus be:

```suggestion
      <span class="sr-only">{{ctx.t('Refresh')}}</span>
```